File: eCopy.Model/Response/ClientResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCopy.Model.Response
{
    public class ClientResponse
    {
        public int ID { get; set; }
        public ApplicationUserResponse ApplicationUser { get; set; }
        public PersonResponse Person { get; set; }
        public int PersonId { get; set; }
        public int ApplicationUserId { get; set; }
    }
}

File: ecopymobile/myapp/lib/model/registrationModel/response/clientResponse.dart
import 'dart:ffi';
import 'package:json_annotation/json_annotation.dart';
import 'package:myapp/model/registrationModel/response/applicationUserResponse.dart';
import 'package:myapp/model/registrationModel/response/personResponse.dart';

part 'clientResponse.g.dart';

@JsonSerializable()
class ClientResponse {
  int? id;
  ApplicationUserResponse? applicationUser;
  PersonResponse? person;
  int? personId;
  int? applicationUserId;

  ClientResponse() {}

  factory ClientResponse.fromJson(Map<String, dynamic> json) =>
      _$ClientResponseFromJson(json);

  /// Connect the generated [_$ClientResponseToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ClientResponseToJson(this);
}
